src/style.css

```diff
@@ -46,7 +46,7 @@ ul li {
 }
 
```

Comment:
**NOTE:**   _**The changes you should apply below are not related to the `style.css file` but are related to your `index.js` file.**_ **I Just Used this part to be able to add inline comments.**
**Please, refer to your `index.js` file to implement the changes required** :pray: 

**[REQUIRED CHANGES]**

- [ ] Hey, according to the project requirement for [To Do list: interactive list](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m2_interactive_list.md), you're yet to add the requirement stated below:

- Add a new JavaScript file and import it as a module:    
    - it will contain methods related to the status updates (`completed`: `true` / `false`).
   According to your implementation, you have only one `javascript file`. Kindly add a new folder called `module`, then add a function called `statusUpdate` which takes up the responsibility of updating the checkbox. if checked[true] updates the task to `completed`, else ...........
You could move this [function](https://github.com/Yilikal250/webpack-demo/blob/628eddaf09d65b594c9a5c961bceb611f3b0cb06/src/index.js#L77-L84) in the file, then update it.

- Please, kindly remove commented code, it makes your code base not clean :pray: